chef/cookbooks/hyperv/recipes/openstack_install_ceilometer.rb

```diff
@@ -24,3 +24,10 @@
   EOH
   not_if { ::File.exists?("#{node[:openstack][:ceilometer][:installed]}") }
 end
+
+utils_line "ensure correct python path in shebang in ceilometer" do
+  file "#{node[:python][:scripts]}\\#{node[:service][:ceilometer][:name]}-script.py"
+  regexp /\A#!.*/
```

Comment:
Lint/AmbiguousRegexpLiteral: Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.